content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -158,6 +173,24 @@ $ helm template $HOME/istio-fetch/istio --name istio --namespace istio-system \
 
 {{% /tab %}}
 
+{{% tab name="remote" cookie-value="remote" %}}
+
+{{< text bash >}}
+$ helm template $HOME/istio-fetch/istio --name istio --namespace istio-system \
+      --values $HOME/istio-fetch/istio/values-istio-remote.yaml | kubectl apply -f -
+{{< /text >}}
+
+{{% /tab %}}
+
+{{% tab name="sds" cookie-value="sds" %}}
+
+{{< text bash >}}
+$ helm template $HOME/istio-fetch/istio --name istio --namespace istio-system \
+      --values $HOME/istio-fetch/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+{{< /text >}}
+
+{{% /tab %}}
+
 {{< /tabset >}}
 
 ### Option 2: Install with Helm and Tiller via `helm install`
```

Comment:
My problem with Option #2 and the way it's laid out is that certain commands like the tiller command, `kubectl apply -f @install/kubernetes/helm/helm-service-account.yaml@` require that one has downloaded the Istio release. There is no way to get the yaml file from the helm repository (at least that's what I was told yesterday).

So, we need instructions or a pointer to that: https://preliminary.istio.io/docs/setup/kubernetes/download/

That then loads to the question of whether we want to use the downloaded repository or the helm repository for the commands. We would need to put the various `fetch` commands in this section, or I think we should just update the commands in the tab to use the local repo.

They would be on the order of:

`helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system`

and in the tabs:
`helm install install/kubernetes/helm/istio --name istio --namespace istio-system`

`helm install install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-demo.yaml`

`helm install install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-minimal.yaml`

`helm install install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-remote.yaml`

Reply:
Although I do see that using the `fetched` would possibly limit the number of changes. Just more difficult getting tiller ready. I recall we used to have a kubectl apply of the file contents pasted in the doc.

Reply:
I put it in the doc for now, but we should come up with a better solution in the future. Note that users will need to download Istio to install the `istioctl` CLI, so we need a better way for that too if we want to be able to use Istio without downloading the release.